syz-ci/manager.go

```diff
@@ -256,6 +256,17 @@ loop:
 	log.Logf(0, "%v: stopped", mgr.name)
 }
 
+func (mgr *Manager) archiveHeadCommit(commit string) {
+	if mgr.cfg.GitArchive == "" || mgr.mgrcfg.DisableGitArchive {
+		return
+	}
+	log.Logf(0, "%v: archiving commit %s", mgr.name, commit)
```

Comment:
Since this happens after every build, the log line looks mostly excessive.
(I assume we already logged the commit used for the build, if not, we should log it where we log successful build)